package.json

```diff
@@ -27,6 +27,7 @@
     "@types/react-dom": "^18.0.9",
     "axios": "^1.3.1",
     "body-parser": "^1.20.1",
+    "classnames": "^2.3.2",
```

Comment:
We already have clsx installed, will that work rather than adding a new dependency?